app/models/manageiq/providers/redhat/infra_manager/vm/operations/snapshot.rb

```diff
@@ -1,4 +1,18 @@
 module ManageIQ::Providers::Redhat::InfraManager::Vm::Operations::Snapshot
+  extend ActiveSupport::Concern
+
+  included do
+    supports :snapshots do
+      unless ext_management_system&.supports_snapshots?
+        if ext_management_system
```

Comment:
there is [`supports :control`](https://github.com/ManageIQ/manageiq/blob/91d5c94c112abf7bcc3b73f743865f7ac682b3ca/app/models/vm_or_template/operations.rb#L99), maybe this suits here.

```ruby
supports :snapshots do
  if supports_control?
    unsupported_reason_add(:snapshots, ext_management_system.unsupported_reason(:snapshots)) unless ext_management_system.supports_snapshots?
  else
    unsupported_reason_add(:snapshots, unsupported_reason(:control))
  end
end